pyproject.toml

```diff
@@ -14,6 +14,7 @@ dynaconf = "^3.2.4"
 tzdata = "^2023.3"
 # this is used in fetch.py
 requests = "^2.31.0"
+aiohttp = "^3.9.1"
```

Comment:
It looks like requests doesn't have async support built-in anymore, or I would have just used requests. Maybe I missed something.

Reply:
requests has never had async support afaik